be/src/exec/pipeline/pipeline_driver_queue.cpp

```diff
@@ -110,6 +111,10 @@ size_t QuerySharedDriverQueue::size() {
 
 void QuerySharedDriverQueue::update_statistics(const DriverRawPtr driver) {
     _queues[driver->get_driver_queue_level()].update_accu_time(driver);
+    if (SourceOperator* source_operator = driver->source_operator()) {
```

Comment:
exchange also is source，should check is scan?

Reply:
The `scan_rows` and `scan_bytes` are maintained at `SourceOperator`, and currently only `OlapChunkSource` provide these two statistic, so the `Exchange` has no affect with it